### PR TITLE
Fix #87 - Remove QuickDocs extension

### LINKED ISSUE #87: Remove the quickdocs extension
According to https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#quick-edit (see the section on Quick Docs), you should be able to use Cmd+K or Ctrl+K to get documentation for the bit of html or css that you're mouse is sitting over.  When I try, I see:
```
Unable to load CSS documentation database:  NotFoundmain.js:69
(anonymous function)jquery-2.1.3.min.js:2
n.Callbacks.jjquery-2.1.3.min.js:2 
n.Callbacks.k.fireWithjquery-2.1.3.min.js:2 
n.extend.Deferred.n.each.e.(anonymous function)FileUtils.js:85 
(anonymous function)File.js:112 
(anonymous function)FilerFileSystem.js:194 
(anonymous function)filer.js:6407 
completefiler.js:5615 
(anonymous function)filer.js:4466 
read_directory_datafiler.js:4126 
read_parent_directory_datafiler.js:4126 
read_parent_directory_datafiler.js:4126 
read_parent_directory_datafiler.js:4141 
get_node_from_parent_directory_datafiler.js:7066 
(anonymous function)filer.js:2263 (anonymous function)
```

The QuickDocs JSON is loaded lazily (i.e., only when you need it), and based on this stack, it looks like it's trying to get it directly from the filesystem.  On desktop, with regular Brackets and a real OS filesystem, this will work fine.  However, in the browser with Filer, we won't have that file in the fs at all.

Our options are to download the JSON files (there's probably more than one we need) in the background after the app starts if it's not already there, so that it *will* be there when it's requested.  Right now we are using a RAM-backed filesystem, so it would mean having to do this every time.  Other options include moving these JSON files into the require bundle and loading with the `text` plugin, or removing this feature altogether.

The `css.json` file is 390K.  My vote is to rip this out, unless someone can figure out a solution.

### PR DESCRIPTION
**Pull request by gideonthomas**
@humphd or @sedge r?

### CODE REVIEW

File: src/utils/ExtensionLoader.js [hunk 1]
```diff
@@ -406,7 +406,6 @@ define(function (require, exports, module) {
                     "QuickOpenJavaScript",
                     "QuickView",
                     "UrlCodeHints",
-                    "WebPlatformDocs",
```
Comment (humphd) [inline src/utils/ExtensionLoader.js]: Keep this change, and remove all others (i.e., don't delete the files, don't remove trailing whitespace, etc.).  We want the fewest changes to Brackets src as possible.

**Comment by humphd** [comment]
r-

**Comment by gideonthomas** [comment]
lol @humphd now?

**Comment by gideonthomas** [comment]
Taken over by https://github.com/humphd/brackets/pull/102

**Pull request closed without merging**